src/state/user.ts

```diff
@@ -45,7 +45,7 @@ export const initialState: UserState = {
   userSwapAdvancedMode: false,
   userPoolAdvancedMode: false,
   userDarkMode: false,
-  gasPriceSelected: GasPrices.Standard,
+  gasPriceSelected: GasPrices.Fast,
```

Comment:
These are a couple places I found throughout the codebase where defaults are set. Are there any others? @hammeiam